frontend/src/lib/utils.tsx

```diff
@@ -1248,7 +1252,7 @@ export function getEventNamesForAction(actionId: string | number, allActions: Ac
 }
 
 export function isPropertyGroup(
-    properties: PropertyGroupFilter | AnyPropertyFilter[] | undefined | AnyPropertyFilter
+    properties: PropertyGroupFilter | PropertyGroupFilterValue | AnyPropertyFilter[] | undefined | AnyPropertyFilter
```

Comment:
```suggestion
    properties?: PropertyGroupFilter | PropertyGroupFilterValue | AnyPropertyFilter[] | AnyPropertyFilter
```

(minor syntactic sugar)